resources/styles/components/course-calendar/month.less

```diff
@@ -122,9 +122,20 @@
 
       &--upcoming {
         cursor: pointer;
-
+        &:after{
```

Comment:
technically it should be `&::after {` <http://www.w3.org/wiki/CSS/Selectors/pseudo-elements/:after> ; )